vfrmap/html/dependencies/dependencies.go

```diff
@@ -1,25 +1,30 @@
-package leafletjs
+package dependencies
 
 import (
+	"log"
 	"net/http"
 )
 
-//go:generate go-bindata -pkg leafletjs -o bindata.go -modtime 1 -prefix "../../../_vendor/leafletjs" "../../../_vendor/leafletjs" "../../../_vendor/leafletjs/images"
```

Comment:
iirc the -modtime 1 was so they generated file always has the same timestamp and thus no git changes if the source files didn't change.

Reply:
the `-modtime` flag does not seem to be available anymore, I used the latest version from https://github.com/jteeuwen/go-bindata

Reply:
`go get -v github.com/jteeuwen/go-bindata/go-bindata@master` the master branch has it